cmd/helpMessages.go

```diff
@@ -30,100 +30,122 @@ Copies source data to a destination location. The supported directions are:
+Upload files and directories by using a SAS token and wildcard (*) characters:
+
   - azcopy cp "/path/*foo/*bar*" "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" --recursive=true
 
-Download a single file using OAuth authentication. Please use 'azcopy login' command first if you aren't logged in yet:
+Download a single file by using OAuth authentication. If you have not yet logged into AzCopy, please use azcopy login command before you run the following command.
+
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]" "/path/to/file.txt"
 
-Download a single file with a SAS:
+Download a single file by using a SAS token:
+
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]" "/path/to/file.txt"
 
-Download a single file with a SAS using piping (block blobs only):
+Download a single file by using a SAS token and then piping the output to a file (block blobs only):
+  
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]" > "/path/to/file.txt"
 
-Download an entire directory with a SAS:
+Download an entire directory by using a SAS token:
+  
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" "/path/to/dir" --recursive=true
 
-A note about wildcards in URLs:
+A note about using a wildcard character (*) in URLs:
 
-The only usage of a wildcard in a URL that is supported is the final, trailing /*, and in the container name when not specifying a blob name.
-If you happen to use the character * in the name of a blob, please manually encode it to %2A to avoid it being treated as a wildcard character.
+There's only two supported ways to use a wildcard character in a URL. 
+You can use one just after the final forward slash (/) of a URL to obtain all of the files in a directory. 
+You can also use one in the name of a container as long as the URL refers only to a container and not to a blob.   
 
-Download the contents of a folder directly to the destination (rather than under a sub-directory):
- - azcopy cp "https://[srcaccount].blob.core.windows.net/[container]/[path/to/folder]/*?[SAS]" "/path/to/dir"
+Download the contents of a directory without copying the containing directory itself.
+ 
+  - azcopy cp "https://[srcaccount].blob.core.windows.net/[container]/[path/to/folder]/*?[SAS]" "/path/to/dir"
 
-Download an entire storage account at once
- - azcopy cp "https://[srcaccount].blob.core.windows.net/" "/path/to/dir" --recursive
+Download an entire storage account.
 
-Download an entire storage account at once with a wildcarded container name
- - azcopy cp "https://[srcaccount].blob.core.windows.net/[container*name]" "/path/to/dir" --recursive
+  - azcopy cp "https://[srcaccount].blob.core.windows.net/" "/path/to/dir" --recursive
+
+Download an entire storage account by using a wildcard symbol (*) in the container name.
+
```

Comment:
Partly our fault on the wording here, but perhaps change "an entire storage account" to " a subset of containers within the storage account"